### PR TITLE
Feature/database

### PR DESCRIPTION
**Pull request by GMahant1**
updated all charts, labels, boxes, to update with darkMode

### CODE REVIEW

File: components/widgets/WeeklyStress.js
Comment (laurenashley) [inline components/widgets/WeeklyStress.js:1]: Line 68 should be `ctx.fillStyle = colours.needle;` to get the right color for the needle for both modes. This works locally for me in this branch.

And do we need line 94-108? Looks like it should be removed.

**Review by laurenashley — COMMENTED** [review]
This looks good to me. What is the icon in the sidebar second form the bottom? Is this the toolbox? If so are we removing this for MVD or keeping it as a talking point for future stretch?

**Comment by kgislason** [comment]
> Is this the toolbox? If

It is logout - doesn't do anything.

**Review by kgislason — APPROVED** [review]

**Comment by kgislason** [comment]
Love the colours!

**Review by Karamvir-Bains — APPROVED** [review]

**Pull request merged**